app/code/Magento/CatalogSearch/Model/Attribute/SearchWeight.php

```diff
@@ -40,21 +40,20 @@ public function __construct(
      *   when an attribute's search weight is changed.
      *
      * @param \Magento\Catalog\Model\ResourceModel\Attribute $subject
-     * @param \Closure $proceed
+     * @param \Magento\Catalog\Model\ResourceModel\Attribute $proceed
      * @param \Magento\Framework\Model\AbstractModel $attribute
      * @return \Magento\Catalog\Model\ResourceModel\Attribute
      *
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
-    public function aroundSave(
+    public function afterSave(
         \Magento\Catalog\Model\ResourceModel\Attribute $subject,
-        \Closure $proceed,
+        \Magento\Catalog\Model\ResourceModel\Attribute $result,
         \Magento\Framework\Model\AbstractModel $attribute
     ) {
         $isNew = $attribute->isObjectNew();
```

Comment:
I think after save it will be always "false", probably that's why we had there "around" plugin. Could you double check it?